convert2rhel/unit_tests/cert_test.py

```diff
@@ -98,30 +98,35 @@ def test_install_cert(self):
         shutil.rmtree(unit_tests.TMP_DIR)
 
 
+def test_remove_cert(caplog, remove_cert_setup):
+    cert_file = system_cert._target_cert_path
+    with open(cert_filename, 'wb') as  cert_file:
+        cert_file.write(b'some content')
+
+    system_cert[sys_cert].remove()
```

Comment:
Is `sys_cert` supposed `cert_file`?

Reply:
sys_cert was some left over code from the old remove_cert_setup() as it returned a dictionary with "sys_cert_instance": sys_cert as one of it's elements, I have already fixed this in https://github.com/oamg/convert2rhel/pull/447/commits/204a0b5aa81e9b95bdebeb71da918e2559cf283f this commit!
Thank you for pointing this out to me!